tcf_website/static/landing/feedbackform.js

```diff
@@ -1,34 +1,40 @@
 import { validateForm } from "../common/form.js";
+import { postToDiscord, sendEmail } from "../common/send_feedback.js";
 
 function submit(event) {
     var form = document.getElementById("feedbackform");
     var valid = validateForm(form);
     if (valid === true) {
```

Comment:
This isn't new in this PR but seems redundant.

Reply:
done

Reply:
Oh, I meant `valid === true` haha.